src/entities/activity/model/hooks/useAnswerSubmittedToast.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from "react"
+
+import { useLocation } from "react-router-dom"
+import { toast } from "react-toastify"
+
+import { useCustomMediaQuery } from "~/shared/utils"
+
+type Props = {
+  text: string
+}
+
+export const useAnswerSubmittedToast = (props: Props) => {
+  const location = useLocation()
+  const { greaterThanSM } = useCustomMediaQuery()
+
+  useEffect(() => {
+    const isAnswersSubmitted = location.state?.isAnswersSubmitted || false
```

Comment:
Well, generally not a reliable source to subscribe to. Better to render toast on more concrete events like function callbacks

Reply:
Done